bundles/org.openhab.binding.velbus/README.md

```diff
@@ -222,10 +349,15 @@ Go to the Items list, select the Item, add a State Description Metadata, and set
 
 ## Full Example
 
+<!--
+FIXME: For an example to be a full example it should probably also show off use of `Thing`
+properties and such
+-->
+
```

Comment:
If you plan to do this, can you create an issue for this instead of adding a FIXME here?

Reply:
I kept the FIXME but now with a reference to the issue that I filed.